src/mainwin.cc

```diff
@@ -1955,7 +1955,7 @@ void MainWindow::actionRenderPreview(bool rebuildParameterWidget)
 
 void MainWindow::csgRender()
 {
-	if (this->root_node) compileCSG(!viewActionAnimate->isChecked());
+	if (this->root_node) compileCSG();
```

Comment:
We used to disable event processes during animation, probably to reduce flicker.
I just noticed that compileCSG() now always processes events. Would that have negative effects on animations? @t-paul any ideas?
I know that this originated in a commit I made 11 months ago; this must have been an oversight..

Reply:
I'm not sure, I found that unused warning a couple of days ago too when looking at some build logs, but was not sure where it's coming from. I don't remember a specific discussion, so oversight is a possibility.